recipes-core/images/snapd-demo-image.bb

```diff
@@ -16,4 +16,4 @@ LICENSE = "MIT"
 inherit core-image
 
 IMAGE_ROOTFS_SIZE ?= "819200"
-IMAGE_ROOTFS_EXTRA_SPACE_append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
+IMAGE_ROOTFS_EXTRA_SPACE:append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
```

Comment:
Nice, I wasn't aware of https://docs.yoctoproject.org/migration-guides/migration-3.4.html#override-syntax-changes